content/testnet.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Setup Rubix Node and Joining the Testnet
+aliases:
+  [
+    "/setup",
+    "/rbt",
+    "/dist",
+    "/start",
+  ]
+geekdocBreadcrumb: false
+weight: 1
+---
+
+Faucet tokens are premined tokens available for users to test and interact with the Rubix testnet.
+
+When a new node joins the testnet, it will have 0 tokens to start with. To help users get started, the Rubix testnet provides a faucet service where users can request premined test tokens. These tokens are not earned through validation but are instead distributed for testing and development purposes.
+
+To request test tokens, users must first create a node in the Rubix testnet and generate a Decentralized Identifier (DID). The DID is then submitted to the faucet, which allocates a fixed number of test tokens to the requesting node.
+
+Test tokens are free and allow users to experiment with the testnet’s functionality, such as performing token transfers, validating token chains, testing smart contracts etc. Unlike mainnet tokens, test tokens do not hold any real-world value and cannot be transferred to the main network.
+
+
+## Pre-requisites  
+
+### 1. Install Go 
+
+Any computer or personal laptop with 8GB of RAM and a decent internet connection can seamlessly run Rubix for token transfers / become validator / mining.
+
+- Download the latest Go binary package for your operating system from [Go's official site](https://golang.org/).  
+- Follow the installation instructions for your OS.  
+- Verify the installation and ensure the Go version is correct:
+
+   ```bash
+   go version
+   ```
+
+### 2. Download and Extract IPFS Kubo (v19)
+
+- Visit [IPFS Kubo releases](https://github.com/ipfs/kubo/releases) and download version 19 suitable for your operating system.
+- Locate the IPFS executable in the extracted files and verify if it has executable permissions.
+
+### 3. Clone the RubixGoPlatform Repository
+
+- Clone the repository:
+    ```bash
+    git clone https://github.com/rubixchain/rubixgoplatform.git
+    ```
+
+- Navigate to the repository directory
+    ```bash
+    cd rubixgoplatform
+    ```
+
+### 4. Build the RubixGoPlatform Executable
+- For Linux : 
+    ```bash
+    make compile-linux
+    ```
+
+- For Windows : 
+    ```bash
+    make compile-windows
+    ```
+
+- For MAC : 
+    ```bash
+    make compile-mac
+    ```
+
+Upon successful compilation, the rubixgoplatform executable will be created in a folder corresponding to your operating system. For example, on Linux, it will be located in the ./linux directory.
+
+### 5. Prepare the Necessary Files
+
+Ensure the following files are in the build directory:
+- rubixgoplatform executable
+- ipfs executable (from Kubo v19)
```

Comment:
Are they supposed to copy it or will it appear automatically?  These are the steps you are supposed to explain in a setup writeup

Reply:
Depends on the folder. If they extract in build file it will appear there automatically, if they extract somewhere else, they will have to copy. That's why I mentioned to ensure things are in the build directory.

Reply:
We should ask them to copy / extract IPFS executable to rubix build folder.  Ensure mean they might think its already supposed to be there